src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -362,22 +388,27 @@
     "useMaterial": {
       "type": "computed",
       "datatype": "bool",
-      "value": "(appTheme == 'material')"
+      "value": "(appTheme == 'material' && templateType == 'default')"
     },
     "useFluent": {
       "type": "computed",
       "datatype": "bool",
-      "value": "(appTheme == 'fluent')"
+      "value": "(appTheme == 'fluent' && templateType == 'default')"
     },
     "useCupertino": {
       "type": "computed",
       "datatype": "bool",
-      "value": "(appTheme == 'cupertino')"
+      "value": "(appTheme == 'cupertino' && templateType == 'default')"
+    },
+    "useLogging": {
```

Comment:
As a fall back for when useDependencyInjection is false, we could use the logging block that's in the unoapp template

Reply:
can you create an issue for that and we can handle the logging change in a new PR